agences-regionales-sante/normandie/2020-03-24.yaml

```diff
@@ -0,0 +1,33 @@
+date: 2020-03-24
+source:
+  nom: ARS Normandie
+  url: https://www.normandie.ars.sante.fr/system/files/2020-03/20200324_CP-point-situation-normandie_0.pdf
+  archive: https://web.archive.org/web/20200324220935/https://www.normandie.ars.sante.fr/system/files/2020-03/20200324_CP-point-situation-normandie_0.pdf
```

Comment:
l'archive is down, j'ai régénéré.
archive: https://web.archive.org/web/20200324230659/https://www.normandie.ars.sante.fr/system/files/2020-03/20200324_CP-point-situation-normandie_0.pdf

Reply:
Corrigé.